src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -39,6 +39,20 @@ using namespace chip;
 using namespace chip::app::Clusters::Thermostat;
 using namespace chip::app::Clusters::Thermostat::Attributes;
 
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace Thermostat {
+static uint32_t sFeatureMap;
+bool AutoSupported      = false;
+bool HeatSupported      = false;
+bool CoolSupported      = false;
+bool OccupancySupported = false;
```

Comment:
Is there a reason these are not static?

But also, this is all assuming the thermostat cluster is only present on one endpoint, right?

Reply:
Cut and paste for not being static.

You do raise and interesting point about the assumption that there is only one thermostat 
endpoint. 

I will move the feature map reads back into the functions that require the information.